data/sql/updates/pending_db_world/rev_1589121527805378200.sql

```diff
@@ -0,0 +1,214 @@
+(7433, 0, 0, 0, 0, 0, 100, 0, 1000, 2000, 16000, 18000, 11, 13443, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Frostsaber Huntress - In Combat - Cast \'13443\''),
+(7433, 0, 1, 0, 2, 0, 100, 1, 5, 30, 0, 0, 11, 15716, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Frostsaber Huntress - Between 5-30% Health - Cast \'15716\' (No Repeat)');
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 7432;
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 7432);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(7432, 0, 0, 0, 11, 0, 100, 0, 0, 0, 0, 0, 11, 30991, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Frostsaber Stalker - On Respawn - Cast \'30991\''),
+(7432, 0, 1, 0, 1, 0, 100, 0, 1200, 1400, 0, 0, 11, 30991, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Frostsaber Stalker - Out of Combat - Cast \'30991\'');
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 7431;
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 7431);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(7431, 0, 0, 0, 0, 0, 100, 0, 1500, 2000, 10000, 11000, 11, 24331, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Frostsaber - In Combat - Cast \'24331\'');
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 7430;
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 7430);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(7430, 0, 0, 0, 0, 0, 100, 0, 1500, 2000, 10000, 11000, 11, 24331, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Frostsaber Cub - In Combat - Cast \'24331\'');
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 10738;
```

Comment:
I think that you put the wrong entry here `10738`, instead of `10199`

Reply:
It should be 10738 for Warchief, dunno how i typed 10199. (https://wotlkdb.com/?npc=10738)
Nice find mate.

Reply:
Grizzle have same AI as well, that's the problem oO
Gonna add it now as well.

Reply:
Edit: Will add for Grizzle in Classic Rare AI, so this should be good for now.
All sorted.